app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -59,7 +59,7 @@ private void nextActivity() {
             LOGGER.info("Sending my profile with name: " + myProfile.getName() + ", public key: "
                     + myProfile.getPublicKey().toString());
             final Contact myProfileAsContact = new Contact(myProfile.getName(), new SimpleKey("simplekey", "73890ien"));
-            this.serviceLocator.getBluetoothHandler().send(myProfileAsContact);
+            this.serviceLocator.getBluetoothHandler().send(this.serviceLocator.getBluetoothHandler().objectToBytes(myProfileAsContact));
```

Comment:
> ...serializable object knows how to serialize itself...

How is that relevant, I don't see anything about serialization here...


> But according to @jverbraeken there had to be only one send with bytes while renaming send and only letting it return bytes so yeah pretty cumbersome but as intended by him

Not sure what you mean there, but why not have two `send()` methods that accept parameters of different types?